spacialgaze-plugins/news.js

```diff
@@ -0,0 +1,86 @@
+/*
+* News System for SpacialGaze
+* Credits: Lord Haji
+*/
+
+'use strict';
+
+let moment = require('moment');
+
+function generateNews() {
+	let newsData , newsDisplay = [];
+	Object.keys(Db('news').object()).forEach (announcement => {
+		newsData = Db('news').get(announcement);
+		newsDisplay.push(`<h4>${announcement}</h4>${newsData[1]}<br /><br />—${SG.nameColor(newsData[0], true)} <small>on ${newsData[2]}</small>`);
+	});
+	return newsDisplay;
+}
+
+function hasSubscribed(user) {
+	if (typeof user === 'object') user = user.userid;
+	if (Db('NewsUnsubscribers').get(toId(user)) === 1) return false;
```

Comment:
Users should by default not be subscribed, they can get the news using a button in the lobby roomintro and then subscribe to automatically get it.